src/Models_Functions/MTV/Compute_M0_T1_OnSPGR.m

```diff
@@ -0,0 +1,83 @@
+function [T1, M0] = Compute_M0_T1_OnSPGR(data, flipAngles, TR, b1Map, roi, verbose)
+%Perform a simple linear-least squares data fit on variable flip angle SPGR data 
+%
+% function [M0, T1] = Compute_M0_T1_OnSPGR(data, flipAngles, TR [, b1Map, roi, verbose])
+% -----------------------------------------------------------
+% INPUTS:
+%   data: width x length x slices x flipAngles matrix
+%   flipAngles: vector of flip angles (in degrees) corresponding to last dimension of 'data'
+%   TR: Repetition time in seconds
+%   b1Map: width x length x slices matrix containing relative flip angle
+%         (i.e. if nominal alpha is 60 and measured alpha is 61, then b1Map = 61/60
+%   roi: width x length x slices binary mask 
+%   verbose: logical - for debugging
+
+
+if ndims(data)<3, data = permute(data(:),[2 3 4 1]); end
+[nX, nY, nZ, nFlip] = size(data);
+nVox = nX*nY*nZ;
+
+if (nargin < 4) || isempty(b1Map)
+    b1Map = ones(nX, nY, nZ);
+end
+
+if nargin<5 || isempty(roi)
+    roi = true(nX, nY, nZ);
+end
+
+if nargin<6
+    verbose = 0;
+end
+ 
+if length(b1Map) ~= length(data(:,:,:,1)), error('B1 size is different from data size'); end
+if ~islogical(roi), roi = roi>0; end
```

Comment:
Why set `roi = roi>0` if `roi` isn't logical? Wouldn't it be better to set `roi = logical(roi)`? I think it reads more clearly from a developer's point of view.

Reply:
Resolved in dff5d5823

Reply:
I always use `>0` because it's less keystrokes than `logical()`. 😀